src/radius_search_filter.cpp

```diff
@@ -52,6 +52,11 @@ bool laser_filters::RadiusSearchFilter::update(
     }
 
     int counter = 0;
+
+    // adaptive threshold based on input_scan range value
+    // the further the point, the larger the radius search is set
+    double cur_threshold = ((int)(input_scan.ranges[i]) + 1) * threshold_ratio_;
```

Comment:
+ 1?  I think we may want to change the filter to be squared_distance (x,y) instead of laser scan distance.  Can discuss more next week, but using the scan distance delta doesn't seem correct (if we expanded the neighbor search it could think that two points are within the radius when they really aren't (10 degrees away from one another but same distance).